project/apps/blog/src/main.ts

```diff
@@ -12,6 +12,9 @@ async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   const globalPrefix = 'api';
   app.setGlobalPrefix(globalPrefix);
+
+  app.useGlobalPipes(new ValidationPipe({ transform: true }));
+  
   const port = process.env.PORT || 3000;
```

Comment:
Нужно вынести числа в константах. Это одно критерий
Так же : [Б10.](https://up.htmlacademy.ru/nodejs-2/9/criteries#b10) Список констант идёт перед основным кодом.

```suggestion
  const port = process.env.PORT |||| DEFAULT_PORT;
```